main.py

```diff
@@ -1,7 +1,39 @@
+from django.contrib.contenttypes.fields import GenericRel  # noqa: F401
+
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    genres = ["Western", "Action", "Dramma"]
+
+    for name in genres:
+        Genre.objects.create(
+            name=name
+        )
+    actors = [
+        ("George Klooney"),
+        ("Kianu Reaves"),
+        ("Scarlett Keegan"),
+        ("Will Smith"),
+        ("Jaden Smith"),
+        ("Scarlett Johansson"),
+    ]
+
+    for actor in actors:
+        first_name, last_name = actor.split(" ")
+        Actor.objects.create(
+            first_name=first_name,
+            last_name=last_name
+        )
+    Genre.objects.filter(name="Dramma").update(name="Drama")
+    Actor.objects.filter(last_name="Klooney").update(last_name="Clooney")
+    Actor.objects.filter(first_name="Kianu").update(
+        first_name="Keanu", last_name="Reeves"
+    )
+    Genre.objects.filter(name="Action").delete()
+    Actor.objects.filter(first_name="Scarlett").delete()
+    return Actor.objects.filter(last_name="Smith").order_by("first_name")
```

Comment:
The function `main` is expected to return a `QuerySet`, which it does. Ensure that the Django ORM is properly initialized before calling this function to avoid runtime errors.